test/kubernetes/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM dockerhub.azk8s.cn/library/golang:1.12.14
```

Comment:
Maybe provide an ARG for the location of dockerhub instead of hard-coding this to the mirror in China?

Reply:
Right. I shall remove the prefix when this pr is done.